index.js

```diff
@@ -143,7 +137,13 @@ Liftoff.prototype.buildEnvironment = function(opts) {
     modulePath: modulePath,
     modulePackage: modulePackage || {},
     configFiles: configFiles,
+    // forcedFlags: forcedFlags,
   };
+
+  // TODO: Does the "prepare" phase allow us to get rid of this normalization?
```

Comment:
@sttk thoughts on this?

Reply:
@sttk I'm thinking that `forcedFlags` should only be an option on launch.  The `execute` command can take `env` and `flags` arguments before the callback.  I'll make this change.